### ISSUE TITLE
Issues running nala after instalation.

### ISSUE THREAD

**Issue by Peterkal2112** [volitank/nala#39]
Hello,

I am running Debian GNU/Linux 11 (bullseye) aarch64 on Raspberry Pi 3 Model B Rev 1.2

I installed nala using [volian-scar](https://gitlab.com/volian/nala/-/wikis/Installation#volian-scar)

I have it installed:

```
nala --version
0.180.0
```

but after trying to use it I getting this error:

```
nala upgrade
usage: nala [-h] [-d] [--version] {init,generate_mocks,wrap_internal_symbols,include_dir,c_sources,cat} ...
nala: error: argument subcommand: invalid choice: 'upgrade' (choose from 'init', 'generate_mocks', 'wrap_internal_symbols', 'include_dir', 'c_sources', 'cat')
```

Can somebody help me with this to get it work ?

**Comment by volitank** [comment]
Yes, this one is presumably because you first tried to install nala with pip, which exists and is not the nala package manager.

try `pip uninstall nala`

By the way there will be some changes to the repositories coming up. Not a big deal and should be mostly automated since you're using the .debs, but wanted you to be aware. I'm going to migrate folks off the scar repository and onto a new one. And of course update the install instructions

**Comment by volitank** [comment]
This GitHub repository is a mirror and issues are not tracked here.

If this is still an issue, please report it on the canonical GitLab repository:
https://gitlab.com/volian/nala/-/issues

Thanks!
